### PR TITLE
Update linux.md

### PR DESCRIPTION
**Pull request by tenaciousdlg**
Removed the wget method of installing cloudflared and instead pointed to the install docs which show how to install it via a package manager or a lower level package manager. Also added pre-requisite criteria before running the commands to install `cloudflared` as a service.

### CODE REVIEW

**Comment by abelinkinbio** [comment]
This one looks good to me 👍 

cc: @abracchi-tw @ranbel

File: content/cloudflare-one/connections/connect-apps/run-tunnel/as-a-service/linux.md [hunk 1]
```diff
@@ -21,14 +21,9 @@ By default, Cloudflare Tunnel expects all of the configuration to exist in the `
 
 ## Running `cloudflared` as a service
 
-Open a terminal window and run the following command to install the latest version of `cloudflared`:
+Instructions on how to install `cloudflared` for Linux can be found [here](https://developers.cloudflare.com/cloudflare-one/connections/connect-apps/install-and-setup/installation/#linux).
```
Comment (ranbel) [inline content/cloudflare-one/connections/connect-apps/run-tunnel/as-a-service/linux.md]:
```suggestion
```

Comment (ranbel) [reply content/cloudflare-one/connections/connect-apps/run-tunnel/as-a-service/linux.md]: This is Step 1 in the link below.

File: content/cloudflare-one/connections/connect-apps/run-tunnel/as-a-service/linux.md [hunk 2]
```diff
@@ -21,14 +21,9 @@ By default, Cloudflare Tunnel expects all of the configuration to exist in the `
...
 
-```sh
-$ wget https://github.com/cloudflare/cloudflared/releases/download/latest/cloudflared-linux-amd64
-mv ./cloudflared-linux-amd64 /usr/local/bin/cloudflared
-chmod a+x /usr/local/bin/cloudflared
-cloudflared update
-```
+Follow up to step 4 [here](https://developers.cloudflare.com/cloudflare-one/connections/connect-apps/install-and-setup/tunnel-guide/#4-create-a-configuration-file) then proceed with the following. At this point `cloudflared` should be installed, authenticated, and have a `config.yml` file in the `.cloudflared` directory.
```
Comment (ranbel) [inline content/cloudflare-one/connections/connect-apps/run-tunnel/as-a-service/linux.md]:
```suggestion
Complete steps 1 through 4 in the [CLI setup guide](/cloudflare-one/connections/connect-apps/install-and-setup/tunnel-guide/). At this point `cloudflared` should be installed, authenticated, and have a `config.yml` file in the `.cloudflared` directory. 
```

**Review by ranbel — CHANGES_REQUESTED** [review]

**Review by ranbel — APPROVED** [review]

**Pull request merged**